packages/search-spa/src/app/search/search.component.ts

```diff
@@ -110,4 +132,12 @@ export class SearchComponent implements OnInit {
   openFeedbackPanel = (): void => {
     (document as any).querySelector('opc-feedback').toggle();
   };
+
+  onBlur() {
+    console.log('poda');
+  }
+
+  onConsole(event: Event) {
+    console.log(event);
+  }
```

Comment:
```suggestion
```

Reply:
My bad. I did remove this. Not sure how it came back. Changed

Reply:
Resolved